pkg/genericapiserver/server/storage_factory.go

```diff
@@ -30,6 +30,15 @@ import (
 	"github.com/golang/glog"
 )
 
+// Backend describes the storage servers, the information here should be enough
+// for health validations
+type Backend struct {
+	Server string
```

Comment:
Document whether this includes the scheme (http/https), or just the IP/hostname?